chapters/0-introduction.tex

```diff
@@ -0,0 +1,266 @@
+on how to complete each task, and links to further practical resources.
+
+\section{Adopting reproducible analytical practices through code}
+
+Modern quantitative research relies heavily on statistical software tools.
+Many development researchers come from economics and statistics backgrounds
+and often understand code to be a means to an end rather than an output itself.
+We believe that this must change somewhat:
+in particular, we think that development practitioners
+must think about their code and programming workflows
+just as methodologically as they think about their research workflows,
+and think of code and data as research outputs, just as manuscripts and briefs are.
+To this end, DIME now maintains strict portfolio-wide standards
+about how analytical code should be maintained and made accessible
+before, during, and after release or publication.
+This is because we see the code as the ``recipe'' for the analysis:
+it tells others exactly what was done,
+how they can do it again in the future,
+and provides a roadmap and knowledge base for further original work.
+DIME Analytics is responsible for maintaining and enforcing these standards,
+and they are intended to cooperatively improve the quality of the work done by
+every member of the DIME team, never to police or punish them.
+
+Similarly, we assume throughout all of this book
+that you are going to do nearly all of your data work though code.
+It is often \textit{possible} to perform nearly all the relevant tasks
+through the interactive user interface in a statistical software,
+or even through non-specialized software such as Excel.
+However, we strongly advise against it.
+The reason for this is the set of
+transparency, reproducibility, and credibility principles
+that we discuss in Chapter 1.
+Performing every task through written code
+creates a record of every task you performed.
+It also prevents direct interaction
+with the data files that could lead to non-reproducible processes.
+Finally, we have invested a lot of time in developing code as a learning tool:
+the examples we have written and the commands we provide
+are designed to provide a framework for common practice
+across the entire DIME team, so that everyone is able to
+read, review, and provide feedback on the work of others
+starting frmo teh same basic ideas about how various tasks are done.
+
+Most specific code tools have a learning and adaptation process,
+meaning you will become most comfortable with each tool
+only by using it in real-world work.
+To support your process of learning reproducible tools and workflows,
+we reference free and open-source tools wherever possible,
+and point to more detailed instructions when relevant.
+Stata, as a proprietary software, is the notable exception here
+due to its persistent popularity in development economics and econometrics.\sidenote{
+  \url{https://aeadataeditor.github.io/presentation-20191211/\#9}}
```

Comment:
are we removing all non-dime wiki references?

Reply:
in the next PR, yes, leave for now so we can do all at once